dev/com.ibm.ws.security.authentication.builtin/src/com/ibm/ws/security/authentication/internal/jaas/JAASServiceImpl.java

```diff
@@ -313,7 +314,9 @@ protected void modified(Map<String, Object> properties) {
             jaasConfigurationFactory.installJAASConfiguration(jaasLoginContextEntries);
 
             configReady();
+            LoginModuleHelper.setTestJaasService(this);
```

Comment:
I thought this method was only used for unit tests.  If it is used by the runtime then the javadoc must be changed and also consider changing the method name:

```java
    /**
     * FOR UNIT TESTING ONLY!!!!
     *
     * Set the test JAASService.
     *
     * @param service The JAASService to use for unit testing.
     */
     public static void setTestJaasService(JAASService service) {
```